test/openssl/utils.rb

```diff
@@ -139,6 +139,22 @@ def teardown
     # OpenSSL error stack must be empty
     assert_equal([], OpenSSL.errors)
   end
+
+  # Omit the tests in FIPS.
+  #
+  # For example, the password based encryption used in the PEM format uses MD5
+  # for deriving the encryption key from the password, and MD5 is not
+  # FIPS-approved.
+  #
+  # See https://github.com/openssl/openssl/discussions/21830#discussioncomment-6865636
+  # for details.
+  def omit_on_fips(&filter_block)
```

Comment:
Note that Rubocop says using `&block` is encouraged rather than `yield` for the explicit block argument in [Rubocop Style Guide](https://github.com/rubocop/ruby-style-guide#explicit-block-argument).